scalafmt-tests/src/test/scala/org/scalafmt/config/ScalafmtConfigTest.scala

```diff
@@ -67,4 +67,20 @@ class ScalafmtConfigTest extends FunSuite {
     assertEquals(config.align, Align.none.copy(stripMargin = true))
   }
 
+  test("dialect override") {
+    val config1 = Config
+      .fromHoconString("""
+        |runner.dialect = scala213
+        |""".stripMargin)
+      .get
+    assert(!config1.runner.getDialect.allowToplevelTerms)
+    val config2 = Config
+      .fromHoconString("""
+        |runner.dialectOverride.allowToplevelTerms = true
```

Comment:
Is it only needed for toplevel terms ? Maybe we should have a dialect for scripts instead?

Reply:
not at all, this is just a test to show that override works.